compose.yaml

```diff
@@ -0,0 +1,64 @@
+name: chainloop-lab
```

Comment:
I am a little bit concerned that by making this a default compose setup it might be interpreted as anything closer to production or even development. It's not suited for any of those purposes.

I am wondering if instead we should have this compose.yaml file and have it inside devel as `labs.compose.yaml` so it doesn't get executed by mistake.